packages/zosfiles/__tests__/__integration__/cli/copy/ds/__snapshots__/cli.files.copy.ds.integration.test.ts.snap

```diff
@@ -0,0 +1,115 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Copy Data Set should display the help 1`] = `
+"================Z/OS FILES COPY DATA SET HELP===============
+
+ COMMAND NAME
+ ------------
+
+   data-set | ds
+
+ DESCRIPTION
+ -----------
+
+   Copy a data set to another data set
+
+ USAGE
+ -----
+
+   zowe zos-files copy data-set <fromDataSetName> <toDataSetName> [options]
+
+ POSITIONAL ARGUMENTS
+ --------------------
+
+   fromDataSetName		 (string)
+
+      The name of the data set that you want to copy from
+
+   toDataSetName		 (string)
+
+      The name of the data set that you want to copy to
```

Comment:
Is it required that the "to" data set be already allocated? If so, might be good to mention that here.